agbenchmark/challenges/code/d3/data.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "TestCreateSimpleWebServer",
+  "category": ["code"],
+  "task": "Build a basic web server that responds to a GET localhost:8079/health with a 200 OK. Deploy this web server locally at the port 8079. ",
+  "dependencies": [],
+  "ground": {
+    "answer": null,
```

Comment:
Writing something simple like 'GET localhost:8079/health responds with a 200 OK' is better then nothing